src/backend/package.json

```diff
@@ -71,5 +78,13 @@
     ],
     "coverageDirectory": "../coverage",
     "testEnvironment": "node"
-  }
+  },
+  "lint-staged": {
+    "*.{js,ts,tsx, jsx}": [
+        "eslint --quiet --fix"
```

Comment:
Couple of inputs here - @binggaofw 

Can we get rid of `--quite` - I want devs to view warnings as well.
Also, let's not fix it; Let it run eslint for you while staging and fail if there are issues - and let developer fix them. In addition, you can create a lint:fix script to solve that.

Reply:
done